### PR TITLE
MOB-687: Using Custom tabs to authenticate the user

### PR DESCRIPTION
**Pull request by matir91**
## Description
This makes the SDK in compliance with Google's new policies regarding mobile authentication

## Changes proposed in this request:
* Removed all code related to handling login through WebView 
* Removed loading indicator and error page from web view. The browser which implement custom tabs shows its own error page in case of missing internet connection. It also has its own way to let the user know that a request is being loaded.

### CODE REVIEW

File: me.id.webverify/app/build.gradle [hunk 1]
```diff
@@ -6,6 +6,9 @@ android {
 
     defaultConfig {
         applicationId "me.id.meidwebverify"
+        manifestPlaceholders = [
+                idmeAuthRedirectScheme: "" // todo: add the redirect scheme
```
Comment (bryant1410) [inline me.id.webverify/app/build.gradle]: Is the indentation right?

File: me.id.webverify/gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Thu Dec 08 14:56:59 UYT 2016
+#Wed Mar 15 12:25:03 UYT 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.2.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
```
Comment (bryant1410) [inline me.id.webverify/gradle/wrapper/gradle-wrapper.properties]: You can use 3.4.1 now.

File: me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/helper/CustomTabsHelper.java [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  private static final String STABLE_PACKAGE = "com.android.chrome";
+  private static final String BETA_PACKAGE = "com.chrome.beta";
+  private static final String DEV_PACKAGE = "com.chrome.dev";
+  private static final String LOCAL_PACKAGE = "com.google.android.apps.chrome";
+  private static final String ACTION_CUSTOM_TABS_CONNECTION = "android.support.customtabs.action.CustomTabsService";
+
+  public static CustomTabsIntent getCustomTabIntent(Context context, String url) {
+    CustomTabsIntent.Builder builder = new CustomTabsIntent.Builder();
+    //noinspection deprecation
+    builder.setToolbarColor(context.getResources().getColor(R.color.idme_blue_dark));
```
Comment (bryant1410) [inline me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/helper/CustomTabsHelper.java]: Chain them man, and maybe don't use the `builder` aux variable.

File: me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/helper/CustomTabsHelper.java [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+        }
+        if (filter.countDataAuthorities() == 0 || filter.countDataPaths() == 0) {
+          continue;
+        }
+        if (resolveInfo.activityInfo == null) {
+          continue;
+        }
+        return true;
+      }
+    } catch (RuntimeException e) {
```
Comment (bryant1410) [inline me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/helper/CustomTabsHelper.java]: Why ignoring here?

File: me.id.webverify/webverifylib/src/main/res/values/style.xml [hunk 5]
```diff
@@ -3,5 +3,8 @@
     <style name="IDmeSdkAppTheme" parent="Theme.AppCompat.NoActionBar">
         <item name="colorPrimary">@color/idme_blue</item>
         <item name="colorPrimaryDark">@color/idme_blue_dark</item>
+        <item name="android:background">@android:color/white</item>
+        <item name="android:windowBackground">@android:color/white</item>
```
Comment (bryant1410) [inline me.id.webverify/webverifylib/src/main/res/values/style.xml]: alpha all here :smile:

File: me.id.webverify/app/build.gradle (hunk 1 above)
Comment (matir91) [inline me.id.webverify/app/build.gradle]: Yes, I think so

File: me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/helper/CustomTabsHelper.java (hunk 3 above)
Comment (matir91) [inline me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/helper/CustomTabsHelper.java]: haha yes!

File: me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/helper/CustomTabsHelper.java (hunk 4 above)
Comment (matir91) [inline me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/helper/CustomTabsHelper.java]: Yes, the os will decide the right package

Comment (bryant1410) [inline me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/helper/CustomTabsHelper.java]: comment it man, and call it `ignored`

**Pull request merged**